frontend/pages/investor/[id]/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { FormattedMessage } from 'react-intl';
+
+import { loadI18nMessages } from 'helpers/i18n';
+
+import ProfileHeader from 'containers/profile-header';
+import SDGs, { SDGType } from 'containers/sdgs';
+import TagsGrid, { TagsGridRowType } from 'containers/tags-grid';
+import { SocialType } from 'containers/website-social-contact';
+
+import Head from 'components/head';
+import LayoutContainer from 'components/layout-container';
+import { StaticPageLayoutProps } from 'layouts/static-page';
+import { PageComponent } from 'types';
+
+export async function getStaticProps(ctx) {
+  return {
+    props: {
+      intlMessages: await loadI18nMessages(ctx),
+    },
+  };
+}
+
+export async function getStaticPaths() {
+  return {
+    paths: [],
+    fallback: true,
+  };
+}
+
+const InvestorPage: PageComponent<{}, StaticPageLayoutProps> = (props) => {
+  const intl = useIntl();
+  // const { query } = useRouter();
+  // const { id } = query;
+
+  const aboutInfo: {
+    text: string;
+    website: string;
+    social: SocialType[];
+    contact: string;
+  } = {
+    text: 'NESsT finances enterprises that generate dignified employment and sustain the planet. Since 1997, we have financed 200 enterprises that have sustained jobs for 70,000 individuals and improved 670,000 lives. We achieve our mission through an Incubator that provides pre-seed capital and business services to improve investment readiness; and the Enterprise Fund that deploys seed-state loans to enterprises seeking capital to scale',
+    website: 'https://www.site.com',
+    social: [
+      { id: 'linked-in', url: 'https://www.linkedin.com' },
+      { id: 'twitter', url: 'https://www.twitter.com' },
+      { id: 'facebook', url: 'https://www.facebook.com' },
+      { id: 'instagram', url: 'https://www.instagram.com' },
+    ],
+    contact: 'LoÏc Comolli',
+  };
+
+  const tagsGridRows: TagsGridRowType[] = [
+    {
+      title: 'Invests in',
+      type: 'category',
+      tags: [
+        { id: 'tourism', title: 'Tourism & Recreation' },
+        { id: 'production', title: 'Non-timber forest production' },
+        { id: 'agrosystems', title: 'Sustainable agrosystems' },
+        { id: 'forestry', title: 'Forestry & agroforestry' },
+      ],
+    },
+    {
+      title: 'Ticket size',
+      tags: ['US$50k', '$50k - $500k', '$500k - $1M'],
+    },
+    {
+      title: 'Instrument size',
+      tags: ['Grand', 'Loan'],
+    },
+    {
+      title: 'Impact they invest on',
+      tags: ['Biodiversity', 'Community'],
+    },
+  ];
+
+  const sdgsItems: SDGType[] = [
+    { id: 'no-poverty', title: 'No poverty' },
+    { id: 'gender-equality', title: 'Gender equality' },
+    { id: 'decent-work', title: 'Decent work and economic growth' },
+    { id: 'reduced-inequalities', title: 'Reduced inequalities' },
+    { id: 'climate-action', title: 'Climate action' },
+    { id: 'life-on-land', title: 'Life on land' },
+  ];
+
+  return (
+    <>
+      <Head title="Investor Profile" />
```

Comment:
Shouldn't we have the name of the investor here? I would also add a description based on the description of the investor.